plasma_framework/contracts/src/exits/payment/controllers/PaymentProcessStandardExit.sol

```diff
@@ -64,9 +68,12 @@ library PaymentProcessStandardExit {
         self.framework.flagOutputFinalized(exit.outputId, exitId);
 
         // we do not want to block a queue if bond return is unsuccessful
-        bool successBondReturn = SafeEthTransfer.transferReturnResult(exit.exitTarget, exit.bondSize, self.safeGasStipend);
-        if (!successBondReturn) {
-            emit BondReturnFailed(exit.exitTarget, exit.bondSize);
+        if (exit.bondSize > exit.bountySize) {
```

Comment:
Wouldn't it better to ensure that bond size is greater than bounty size when ever those values are set or updated in the router?

Reply:
Yes, doing it here: 
while setting - https://github.com/omgnetwork/plasma-contracts/pull/707/files#diff-a7dc70921b2cc940820c62c3ca5bcc0b715a92b16cd75d8956c86292bb7bd063R39
while updating - https://github.com/omgnetwork/plasma-contracts/pull/707/files#diff-a7dc70921b2cc940820c62c3ca5bcc0b715a92b16cd75d8956c86292bb7bd063R94

The check here is mostly to not attempt a bond return if bond_size is equal to bounty_size

Reply:
Yeah just saw that after I made the comment. Still think that bond size should always be greater than bounty size and setting them to equal amounts should be rejected in any case.

Reply:
Shouldn't we allow the bond size to be equal the bounty size? Given we do this- `The bond should cover the cost of either challenging or processing the exit. which of those has a higher cost and set the bond to that.`
Is there a problem that is introduced due to this?

Reply:
I guess it's fine.